code/datums/autocells/explosion.dm

```diff
@@ -57,15 +54,12 @@
 	// For stat tracking and logging purposes
 	var/datum/cause_data/explosion_cause_data
 
-	// Workaround to account for the fact that this is subsystemized
-	// See on_turf_entered
-	var/list/atom/exploded_atoms = list()
-
 	var/obj/effect/particle_effect/shockwave/shockwave = null
 
 // If we're on a fake z teleport, teleport over
 /datum/automata_cell/explosion/birth()
 	shockwave = new(in_turf)
+	QDEL_IN(shockwave, 0.2 SECONDS)
```

Comment:
I haven't looked in great detail but I don't think you want to tie in explosion lifecycle to time, because the whole contentious point with them and the reason this PR exists to begin with is that there is a noticeable drift between subsytem ticks and world time

Reply:
this is the shockwave effect though, not the explosion itself